src/cljs/org/broadinstitute/firecloud_ui/common/icons.cljs

```diff
@@ -5,30 +5,33 @@
     [org.broadinstitute.firecloud-ui.utils :as utils]
     ))
 
-(def ^:private icon-keys {:activity ""
-                          :angle-left ""
-                          :angle-right ""
-                          :document ""
-                          :gear ""
-                          :information ""
-                          :locked ""
-                          :pencil ""
-                          :plus ""
-                          :remove ""
-                          :search ""
-                          :share ""
-                          :status-done ""
-                          :status-warning ""
-                          :status-warning-triangle ""
-                          :trash-can ""
-                          :view-mode-list ""
-                          :view-mode-tiles ""
-                          :x ""})
+;; From https://design.google.com/icons/
+(def ^:private icon-keys
+  {:angle-left "keyboard_arrow_left"
+   :angle-right "keyboard_arrow_right"
+   :settings "settings"
+   :lock "lock"
+   :unlock "lock_open"
+   :edit "mode_edit"
+   :clone "content_copy"
+   :add "add"
+   :delete "delete"
+   :remove "remove_circle"
+   :done "done"
+   :cancel "cancel"
+   :warning-triangle "warning"
+   :error "error"
+   :search "search"
+   :share "share"
+   :library "import_contacts"
+   :publish "publish"
+   :catalog "local_offer"})
```

Comment:
Now all you have to do to introduce a new icon is find the one you like and register it here.